src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_PROPERTY_SIZE = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            default:
+                return getCircle(FigureType.RANDOM);
```

Comment:
better to have `default` at the end of the switch statement, as a last resort by style wise